src/base/Prometheus.js

```diff
@@ -31,6 +32,12 @@ register.registerMetric(ruleGauge)
 // Format community trust from config
 const trustedCommunities = async (communities) => {
 	let rawResults = []
+	const CachedCommunities = new Map()
+	const getOrFetchCommunity = async (communityid) => {
+		if (CachedCommunities.get(communityid)) return CachedCommunities.get(communityid)
```

Comment:
Okay, cache is understandable. I'll set it to clear every 15 minutes because that seems as a reasonable amount for stuff to not be changed. If anything happens, you'll just wait for a bit.

> And why does the bot include stats tracking, would it not be more efficient and logical to have it as part of the backend.
Hmm. I guess there wouldn't need to be requests to get communities or rules. Migrated to the backend.